modules/ROOT/pages/index.adoc

```diff
@@ -5,25 +5,25 @@ endif::[]
 :keywords: hl7, healthcare, toolkit, hapi, mllp, er7
 :license-info: Enterprise, CloudHub
 
-Mule Healthcare Toolkit is a collection of features that facilitates integration with healthcare systems by providing the tools needed to easily create, read, and transform HL7 messages within Mule.
+Healthcare Toolkit is a collection of features that facilitates integration with healthcare systems by providing the tools needed to easily create, read, and transform HL7 messages within Mule.
```

Comment:
Please remove "within Mule" -- not needed and probably the writer meant Studio and in Anypoint Platform, which contradicts with the idea of "Mule" being on-premises.